File: Projects_Javascript - Copy/JS/project.js
function Project() {
    this.name = "";
    this.allocation = 0;
    this.startdate = "";
    this.enddate = "";
    this.status = "Open";



}

Project.prototype.addProject = function () {
    this.name = $('projectname').value;
    this.startdate = $('startdate').value;
    this.enddate = $('enddate').value;
    this.allocation = $('allocation').value;
}

Project.prototype.changeStatus = function (status) {
    this.status = status;
}

Project.prototype.setProject = function (obj) {
    this.name = obj.name;
    this.allocation = obj.allocation;
    this.startdate = obj.startDate;
    this.enddate = obj.endDate;
    this.status = obj.status;
}

File: ProjectsAPI-V2/ProjectsAPI/ProjectDomain/Project.cs
using System;

namespace Domain
{
    public class Project
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int? Allocation { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public string Status { get; set; }
    }
}
